cabot/metricsapp/api/elastic.py

```diff
@@ -1,11 +1,54 @@
 from elasticsearch import Elasticsearch
+from enum import Enum
+
+
+class SupportedMetrics(Enum):
```

Comment:
Oh yeah, constants in defs.py sounds much better!

The difference is just whether the metric is expected under value or values in the response. I guess we don't really have to distinguish between them though, we could just look for both "value" and "values" when we parse the json--I thought it'd be good to make sure the response metric is the type we expect, but seems unlikely we'd get weird responses since we're validating queries...